src/java/org/apache/cassandra/cql3/Operator.java

```diff
@@ -324,16 +480,131 @@ public static Operator readFrom(DataInput input) throws IOException
           throw new IOException(String.format("Cannot resolve Relation.Type from binary representation: %s", b));
     }
 
+
     /**
      * Whether 2 values satisfy this operator (given the type they should be compared with).
      */
     public abstract boolean isSatisfiedBy(AbstractType<?> type, ByteBuffer leftOperand, ByteBuffer rightOperand);
 
-    public int serializedSize()
+    public static int serializedSize()
     {
         return 4;
```

Comment:
We could change this to be a byte at some time

Reply:
True but it is always complex with backward compatibility.